apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -28,6 +28,8 @@ import (
 	ironicv1 "github.com/openstack-k8s-operators/ironic-operator/api/v1beta1"
 	keystonev1 "github.com/openstack-k8s-operators/keystone-operator/api/v1beta1"
 	condition "github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/route"
```

Comment:
If we are moving all the route logic here would it potentially make sense to remove common/route from lib-common and instead just host it within this operator?

Reply:
yes we can do that as a follow up, otherwise we'd have to first migrate all service operators before we can land other things in lib-common